rebar.config

```diff
@@ -12,8 +12,22 @@
              {pretty_print, erl_pp},
              {preprocess, true}]}.
 
+{deps_error_on_conflict, true}.
+
 {deps, [
-    {eredis,"1.*", {git, "https://github.com/Nordix/eredis.git", {tag, "v1.4.1"}}},
-    {ecredis_crc16, ".*", {git, "git@github.com:HalloAppInc/ecredis-crc16.git", {branch, "master"}}},
-    {lager, ".*", {git, "https://github.com/erlang-lager/lager", "3.6.10"}}
+    {eredis, "1.*", {git, "https://github.com/Nordix/eredis.git", {tag, "v1.4.1"}}},
+    {ecredis_crc16, ".*", {git, "https://github.com/HalloAppInc/ecredis-crc16.git", {branch, "master"}}},
+    {lager, ".*", {git, "https://github.com/erlang-lager/lager", {tag, "3.6.10"}}}
     ]}.
+
+%% see README here: https://github.com/blt/port_compiler
```

Comment:
I saw the readme. I could not understand the significance. Can you please comment.

Reply:
the large block that comes below this comment is explained in the README. Basically it's for a rebar2 dependency that uses the port compiler for rebar2. I wanted to include a comment of how/why that block was inserted, in case this is being reviewed in the future

Reply:
Can I please request you to add a comment to explain the significance of the next section.